server/update/model.ts

```diff
@@ -0,0 +1,93 @@
+import type {Types} from 'mongoose';
+import {Schema, model} from 'mongoose';
+import type {User} from '../user/model';
+// import type {Project} from '../project/model';
+
+/**
+ * This file defines the properties stored in a User
+ * DO NOT implement operations here ---> use collection file
+ */
+
+// Type definition for Update on the backend
+export type Update = {
+  _id: Types.ObjectId; // MongoDB assigns each object this ID on creation
+  authorId: Types.ObjectId;
+  dateCreated: Date;
+  dateModified: Date;
+  status: string;
+  summary: string;
+  details: string;
+  todos: string;
+  blockers: string;
```

Comment:
To clarify, will todos/blockers still be arrays of strings? The validators imply that's still the case, but the schema does not match that.

Reply:
oops hahaha I totally forgot to change that. fixed!